src/presentation/ui/component/form/textArea.templ

```diff
@@ -29,3 +29,52 @@ templ TextArea(
 		}
 	</div>
 }
+
+script TextAreaReadOnlyLocalState() {
+	document.addEventListener('alpine:init', () => {
+		Alpine.data('textAreaReadOnly', () => ({
+			copyToClipboard(elementId) {
+				const textInputElement = document.getElementById(elementId);
+				textInputElement.select();
+				navigator.clipboard.writeText(textInputElement.value);
+			}
+		}))
+	})
+}
+
+templ TextAreaReadOnly(
+	id, label, bindValuePath string,
+	denseMode bool,
+) {
+	<!-- TextAreaReadOnly -->
+	@TextAreaReadOnlyLocalState()
+	<div class="relative w-full" x-data="textAreaReadOnly">
+		<textarea
+			id={ id }
+			name={ id }
+			if bindValuePath != "" {
+				x-model:value={ bindValuePath }
+			}
+			placeholder={ label }
+			if denseMode {
```

Comment:
If not using denseMode, please remove, that's not a standard you have to follow.

Reply:
Done!